src/dotnet/APIView/APIViewWeb/Pages/Assemblies/RequestedReviews.cshtml

```diff
@@ -2,155 +2,406 @@
-                <table id="reviews-table" class="table table-sm table-hover m-0" style="width:100%">
-                    <thead>
-                        <tr>
-                            <th scope="col" class="pl-4">Name</th>
-                            <th scope="col" class="border-left pl-3">CreatedBy</th>
-                            <th scope="col" class="border-left pl-3">Last Updated</th>
-                            <th scope="col" class="border-left pl-3">Type</th>
-                            <th scope="col" class="border-left pl-3">Requested on</th>
-                            <th scope="col" class="border-left pl-3">Requested by</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        @if (Model.ActiveAPIRevisions.Any())
-                        {
-                            @foreach (var apiRevision in Model.ActiveAPIRevisions)
+    <div class="col-md-12" id="reviews-tabs-container">
+        <!-- Tab Navigation -->
+        <ul class="nav nav-tabs mt-3" id="reviewTabs" role="tablist">
+            <li class="nav-item" role="presentation">
+                <a class="nav-link active" id="pending-reviews-tab" data-toggle="tab" href="#pending-reviews" role="tab" aria-controls="pending-reviews" aria-selected="true">
+                    Pending Reviews
+                    @if (Model.ActiveAPIRevisions.Any())
+                    {
+                        <span class="badge badge-primary ml-2">@Model.ActiveAPIRevisions.Count()</span>
+                    }
+                </a>
+            </li>
+            <li class="nav-item" role="presentation">
+                <a class="nav-link" id="pending-namespace-tab" data-toggle="tab" href="#pending-namespace" role="tab" aria-controls="pending-namespace" aria-selected="false">
+                    Pending Namespace Approvals
+                    @if (Model.NamespaceApprovalRequestedAPIRevisions.Any())
+                    {
+                        <span class="badge badge-warning ml-2">@Model.NamespaceApprovalRequestedAPIRevisions.Count()</span>
+                    }
+                </a>
+            </li>
+            <li class="nav-item" role="presentation">
+                <a class="nav-link" id="recently-approved-tab" data-toggle="tab" href="#recently-approved" role="tab" aria-controls="recently-approved" aria-selected="false">
+                    Recently Approved
+                    @if (Model.ApprovedAPIRevisions.Any())
+                    {
+                        <span class="badge badge-success ml-2">@Model.ApprovedAPIRevisions.Count()</span>
+                    }
+                </a>
+            </li>
+            <li class="nav-item" role="presentation">
+                <a class="nav-link" id="reviews-without-namespace-tab" data-toggle="tab" href="#reviews-without-namespace" role="tab" aria-controls="reviews-without-namespace" aria-selected="false">
```

Comment:
we should delete this. it might create more confusion than it is worth. Architects are not proactive

Reply:
bc of time, lol